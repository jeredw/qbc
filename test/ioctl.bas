ioctl 1, "hello"
ioctl #1, "yes"
print ioctl$(1)
print ioctl$(#1)
