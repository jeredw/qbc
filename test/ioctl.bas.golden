(program 
  (statement 
    (ioctl_statement ioctl 
      (expr 
        (literal 1)) , 
      (expr 
        (literal "hello")))) \n 
  (statement 
    (ioctl_statement ioctl # 
      (expr 
        (literal 1)) , 
      (expr 
        (literal "yes")))) \n 
  (statement 
    (print_statement print 
      (expr 
        (builtin_function ioctl$ 
          ( 
            (expr 
              (literal 1)) ))))) \n 
  (statement 
    (print_statement print 
      (expr 
        (builtin_function ioctl$ 
          ( # 
            (expr 
              (literal 1)) ))))) \n <EOF>)
